packages/admin/src/Http/Livewire/Concerns/CanNotify.php

```diff
@@ -8,6 +8,6 @@ trait CanNotify
 {
     public function notify(string $status, string $message, bool $isAfterRedirect = false): void
```

Comment:
Well done for keeping the parameter here, removing them would be a breaking change 👍

Reply:
Any feedback for this?

> And would you want to deprecate that isAfterRedirect param? I thought about #[Deprecated] but that is PhpStorm specific. Alternatively we could trigger a deprecation error.

Reply:
Just use a deprecated DocBlock instead.

Reply:
`@deprecated` only works for full methods not for params, I think?